src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -3912,6 +3926,21 @@ internal static TraceJittedMethod MethodComplete(TraceLoadedDotNetRuntime stats,
             return _method;
         }
 
+        /// <summary>
+        /// Handles AllocRequest event for JIT
+        /// </summary>
+        internal static TraceJittedMethod JitAllocComplete(TraceLoadedDotNetRuntime stats, LoaderHeapAllocRequestTraceData data)
```

Comment:
I would recommend calling this method something like `LogJitAllocation` or something along those lines, since it's not actually a completion operation.  `MethodComplete`, for example, is responsible for marking the JIT operation complete, which is where the complete name comes from.